test_util/cluster_api.py

```diff
@@ -187,7 +187,7 @@ def _authenticate(self):
         self.superuser_auth_cookie = r.cookies['dcos-acs-auth-cookie']
 
     def __init__(self, dcos_uri, masters, public_masters, slaves, public_slaves,
-                 dns_search_set, provider, auth_enabled, username, password):
+                 dns_search_set, provider, auth_enabled, username, password, default_os_user='root'):
```

Comment:
I did this so I could avoid editing the vagrant test. Editing so its a required arg